Sources/ToastCenter.swift

```diff
@@ -23,7 +23,7 @@ open class ToastCenter {
     #if swift(>=4.2)
     let name = UIDevice.orientationDidChangeNotification
     #else
-    let name = NSNotification.Name.UIDeviceOrientationDidChange
+    let name = UIDevice.orientationDidChangeNotification
```

Comment:
this change is incorrect, as we're in the #else for older Swift version: don't blindly trust the Migration assistant